### PR TITLE
Cve-2020-8908 Update

### PR DESCRIPTION
**Pull request by pauldevt**

### CODE REVIEW

File: 2020/8xxx/CVE-2020-8908.json [hunk 1]
```diff
@@ -43,7 +43,7 @@
...
+		"value": "A temp directory creation vulnerability exists in all versions of Guava, allowing an attacker with access to the machine to potentially access data in a temporary directory created by the Guava API com.google.common.io.Files.createTempDir(). By default, on unix-like systems, the created directory is world-readable (readable by an attacker with access to the system). The method in question has been marked @Deprecated in versions 30.0 and later and should not be used. For Android developers, we recommend choosing a temporary directory API provided by Android, such as context.getCacheDir(). For other Java developers, we recommend migrating to the Java 7 API java.nio.file.Files.createTempDirectory() which explicitly configures permissions of 700, or configuring the Java runtime's java.io.tmpdir system property to point to a location whose permissions are appropriately configured."
```
Comment (cristina479) [inline 2020/8xxx/CVE-2020-8908.json:46]: Are all versions of Guava affected or only versions > 9.09.15 as stated in the `version` field? Please correct the discrepancy.

Comment (pauldevt) [reply 2020/8xxx/CVE-2020-8908.json:46]: Version 9.09.15 was the first publicly available version of Guava and it effects everything => that.
What would be advisable here with the version value?

Comment (JLLeitschuh) [reply 2020/8xxx/CVE-2020-8908.json:46]: Vuln reporter here: There are no fixed versions of Guava.

**Comment by tony--** [comment]
@cve-team I was glad to see this merged and land on NVD at https://nvd.nist.gov/vuln/detail/CVE-2020-8908
When I reviewed this today, I noticed that there is a minor problem with Configuration 1 in the "Known Affected Software Configurations" section.  Although the CPE string is correct, the text still says "Up to (excluding) 30.0".  I don't see this anywhere in the change you made here, so perhaps this is something that NVD maintains separately?  Should I open a separate issue for tracking this or will you just contact them?
FYI @JLLeitschuh 
Also mention https://github.com/google/guava/issues/4011

**Pull request merged**